test/unit/entities/organization-membership-test.js

```diff
@@ -0,0 +1,56 @@
+import test from 'blue-tape'
+import {cloneMock} from '../mocks/entities'
+import setupHttpMock from '../mocks/http'
+import {wrapOrganizationMembership, wrapOrganizationMembershipCollection} from '../../../lib/entities/organization-membership'
+import {
+  entityWrappedTest,
+  entityCollectionWrappedTest,
+  entityUpdateTest,
+  entityDeleteTest,
+  failingActionTest
+} from '../test-creators/instance-entity-methods'
+
+function setup (promise) {
+  return {
+    httpMock: setupHttpMock(promise),
+    entityMock: cloneMock('organizationMembership')
+  }
+}
+
+test('OrganizationMembership is wrapped', (t) => {
+  entityWrappedTest(t, setup, {
+    wrapperMethod: wrapOrganizationMembership
+  })
+})
+
+test('OrganizationMembership collection is wrapped', (t) => {
+  return entityCollectionWrappedTest(t, setup, {
+    wrapperMethod: wrapOrganizationMembershipCollection
+  })
+})
+
+test('OrganizationMembership update', (t) => {
+  return entityUpdateTest(t, setup, {
```

Comment:
These are the same tests that didn't catch that the update didnt work, right? I thought we wanted to update them to actually test that the update happens? :)

Reply:
@phoebeschmidt exactly. do you think I need to write custom one, not to use this util functions?

Reply:
yeah if these tests aren't successfully testing the functionality, then we should write ones that do